libr/include/sdb/types.h

```diff
@@ -40,9 +41,15 @@
 #define DIRSEP '/'
 #endif
 
+#ifndef _MSC_VER
+#include <unistd.h>
+#else
+#include <io.h>
+#endif
+
 #include <inttypes.h>
 #define ULLFMT "ll"
-#if __SDB_WINDOWS__ && !__CYGWIN__
+#if defined(_MSC_VER) || (__SDB_WINDOWS__ && !__CYGWIN__)
```

Comment:
better set this check at line 33, then __SDB_WINDOWS__ come to 1 and this block is included